### PR TITLE
[ADD] LuciphorAdapter FileNotFound 

### PR DESCRIPTION
**Pull request by oliveralka**
In regard to #4301

Depending on the parameters e.g. "min_num_psms_model". LuciPhor2 will finish processing, but the output file will be empty, which leads to a more or less cryptic error message. 

Additional:
Changes the debug option, which can report external calls from >= 10 to >= 4.

### CODE REVIEW

File: src/tests/topp/THIRDPARTY/third_party_tests.cmake [hunk 1]
```diff
@@ -356,3 +360,10 @@ if (NOT (${SPECTRAST_BINARY} STREQUAL "SPECTRAST_BINARY-NOTFOUND") AND FALSE)
   set_tests_properties("TOPP_SpectrastSearchAdapter_1_out" PROPERTIES DEPENDS "TOPP_SpectrastSearchAdapter_1")
 endif()
 
+if (NOT (${LUCIPHOR_BINARY} STREQUAL "LUCIPHOR_BINARY-NOTFOUND"))
+  ### NOT needs to be added after the binarys have been included
```
Comment (jpfeuffer) [inline src/tests/topp/THIRDPARTY/third_party_tests.cmake]: Copy-paste from crux adapter? I don't even know what it means there. Any idea?

Comment (jpfeuffer) [reply src/tests/topp/THIRDPARTY/third_party_tests.cmake]: But let the tests run through once first before making another change.

**Comment by oliveralka** [comment]
Yes, I did test the Exception locally.

File: src/tests/topp/THIRDPARTY/third_party_tests.cmake (hunk 1 above)
Comment (oliveralka) [reply src/tests/topp/THIRDPARTY/third_party_tests.cmake]: yes, copy and paste error!

**Comment by oliveralka** [comment]
@jpfeuffer: Error seems unrelated: ` Error when uploading file`

**Pull request merged**